core/src/main/java/org/apache/brooklyn/core/mgmt/persist/DeserializingJcloudsRenamesProvider.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.brooklyn.core.mgmt.persist;
+
+import java.util.Arrays;
+
+public class DeserializingJcloudsRenamesProvider extends DeserializingProvider{
+
+    private static DeserializingJcloudsRenamesProvider instance;
+
+    public static DeserializingJcloudsRenamesProvider getInstance(){
+        if (instance == null) instance = new DeserializingJcloudsRenamesProvider();
+        return instance;
+    }
+
+    private DeserializingJcloudsRenamesProvider(){
+        super(Arrays.asList(new ConfigLoader[]{
```

Comment:
Ah, good point. Given that the super-constructor is protected, then that's ok. Could instead use: `super(MutableList.of(new JcloudsProviderRenameConfigLoader()));`

As a side-note, if the constructor were public, then it would be better to take a copy of the list (rather than relying on the caller passing in a mutable list).

If we do continue to rely on the constructor being called with a mutable list (rather than it taking a copy), then probably worth adding a javadoc comment about that parameter.